Classes/Service/Utility.php

```diff
@@ -45,7 +45,9 @@ class Utility implements SingletonInterface
 
     public function __construct()
     {
-        $this->connectionPool = GeneralUtility::makeInstance(ConnectionPool::class);
+        if (version_compare(TYPO3_branch, '8.7', '>=')) {
```

Comment:
Maybe we should go for `if (class_exists(ConnectionPool::class)) {` instead of the version check?

Can be done in a followup